Automattic/jetpack
# Jetpack: use BlockIcon in Placeholder VideoPress component

This PR uses `<BlockIcon />` core component to simplify the way to render the icon in the PlaceholderWrapper VPBlock component.
Follow-up of https://github.com/Automattic/jetpack/pull/25058

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: use BlockIcon in Placeholder VideoPress component

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block editor
* Create a new VPBlock
* Upload a video. Try to see different states of the block until the video player is rendered
* Visually, it should not be any change. Confirm everything works as expected.



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202586735461850
